src/schneider_lab_to_nwb/zempolich_2024/zempolich_2024_nwbconverter.py

```diff
@@ -39,5 +56,61 @@ def temporally_align_data_interfaces(self):
         behavior_file_path = Path(behavior_interface.source_data["file_path"])
         file = read_mat(behavior_file_path)
         cam1_timestamps, cam2_timestamps = file["continuous"]["cam"]["time"]
+        if conversion_options["Behavior"].get("normalize_timestamps", False):
+            starting_timestamp = get_starting_timestamp(mat_file=file)
+            cam1_timestamps -= starting_timestamp
+            cam2_timestamps -= starting_timestamp
         self.data_interface_objects["VideoCamera1"].set_aligned_timestamps([cam1_timestamps])
         self.data_interface_objects["VideoCamera2"].set_aligned_timestamps([cam2_timestamps])
+
+    # NOTE: run_conversion is copy-pasted from neuroconv.NWBConverter and only modified to call
```

Comment:
Not that it detracts from the idea of passing more parameters to the alignment function but it is way less brittle to:

```python
def run_conversion(all_kwargs):
     self.conversion_options = conversion_options
     self.metadata = metadata
     
     super.run_conversion(all_kwargs)
```

Then you can use the attributes inside the `temporally_align_data_interfaces`

This is way less moving pieces than what you have.

Reply:
Thanks! I had a feeling there would be a better way to do this, but for some reason this solution escaped me.